HabitTracker.BatataDoc3/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "HabitTracker.BatataDoc3": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\.NetLearning\\CodeReviews.Console.HabitTracker\\HabitTracker.BatataDoc3"
```

Comment:
If you are hard-coding paths that may not be on users machines then this needs to be a step in the README to change:

I have to make a change to this value to run your code!

![image](https://github.com/user-attachments/assets/4ccaa027-5c41-4fa8-9a3c-b0dd51cd94ab)